### DISCUSSION TITLE
Hide label if cursor is directly within the label area

### DISCUSSION THREAD

**Discussion by MatejBransky** [b0o/incline.nvim#66] [Questions and Answers]
Hello, thanks for the plugin 🙏 , it's really awesome. 🎉 

However, I'm trying to figure out how I could keep the label even when I'm hovering over the first row and the cursor isn't on the label itself. 🤔 

Is it possible in the current version already? 🙂

**Comment by b0o** [comment]
Converted to an issue: #67
